apps/wallet/src/ui/app/pages/swap/utils.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { coinsMap } from '_app/hooks/useDeepBook';
+
+export const DEFAULT_MAX_SLIPPAGE_PERCENTAGE = 0.5;
+export const FEES_PERCENTAGE = 0.03;
+
+export const initialValues = {
+	amount: '',
+	isPayAll: false,
+	quoteAssetType: coinsMap.USDC,
+	allowedMaxSlippagePercentage: DEFAULT_MAX_SLIPPAGE_PERCENTAGE,
+};
+
+export type FormValues = typeof initialValues;
```

Comment:
Same here, this seems unused?

Reply:
They are used in subsequent PR here https://github.com/MystenLabs/sui/pull/14068

Reply:
In the future, break these PRs up so that these land together. This can't be reviewed in isolation.